service/internal/resources.go

```diff
@@ -1,4 +1,4 @@
-// Copyright  OpenTelemetry Authors
+// Copyright 2020 The OpenTelemetry Authors
```

Comment:
Should we skip the year?

Reply:
@bogdandrutu currently addlicense in Makefile adds the year so I was just trying to maintain consistency.

Reply:
I think we removed globally

Reply:
https://github.com/google/addlicense/issues/36 😆

Reply:
> I think we removed globally

@bogdandrutu I'm confused, it's still in a bunch of the file headers so I don't think it's been removed?

Reply:
Do you want me to add -y"" here and in Makefile?

Reply:
Please do in a separate PR :)